src/main.rs

```diff
@@ -625,6 +632,12 @@ async fn get_setup() -> Setup {
         "Knee width (dB) of the dynamic limiter from 0.0 to 10.0. Defaults to 5.0.",
         "KNEE",
     )
+    .optopt(
```

Comment:
Could we do without this feature flag? Because it seems pretty advanced. What would it matter to users of we did or did not drop it?